sessions-core/src/test/java/com/adyen/checkout/sessions/models/TestPaymentMethod.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 2/2/2023.
+ */
+
+package com.adyen.checkout.sessions.models
+
+import android.os.Parcel
+import com.adyen.checkout.components.model.payments.request.PaymentMethodDetails
+
+class TestPaymentMethod(
```

Comment:
Should this be moved to a more common module like `:checkout-core` or `test-core`? Or should it be `internal`?

Reply:
`test-core` wouldn't work because this class extends a class from `components-core`, I think `internal` + duplication would be okay since it doesn't have that much logic anyway.

Reply:
Agree 👍